sagemaker-pyspark-sdk/tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = clean,flake8,docs,py{27,35,36},stats
+envlist = clean,flake8,docs,py{27,36},stats
```

Comment:
you also probably didnt want to commit this.

Reply:
We need this to speed up integ tests. But I see your concern of 5 commits in PR.

Let me try rebasing to master. If not permitted, we can discuss on what to do next.